src/main/java/teammates/ui/controller/FeedbackSubmissionEditSaveAction.java

```diff
@@ -239,27 +245,45 @@ private boolean isExistingResponseValid(FeedbackResponseAttributes response) {
         return existingResponsesId.contains(response.getId());
     }
 
-    private void saveResponse(FeedbackResponseAttributes response)
-            throws EntityDoesNotExistException {
+    private void collectResponses(FeedbackResponseAttributes response, List<FeedbackResponseAttributes> responsesToSave,
```

Comment:
This name is misleading. It collects only one response per call. `addToPendingResponses` or something like that?